docs/content/docs/2.components/calendar.md

```
@@ -275,6 +334,26 @@ name: 'calendar-external-controls-example'
 ---
 ::
 
+### As a month picker
+
+Use `type="month"` with a [Popover](/docs/components/popover) to create a month picker with a 3-year window.
+
+::component-example
+---
+name: 'calendar-month-picker-example'
```

markdownlint-cli2 (0.21.0)

[warning] Heading style
Expected: atx; Actual: setext

(MD003, heading-style)